02Lesson/01Project/index.html

```diff
@@ -1,9 +1,61 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-</head>
-<body>
+  <head>
+    <title>My Bio</title>
+    <meta charset="utf-8">
+  </head>
+  <body>
+   <section>
```

Comment:
Indentation: Always use 2 spaces consistently, here is only one space which might lead to errors further down in the code.